src/main/java/dk/kb/storage/util/DsStorageClient.java

```diff
@@ -208,13 +403,21 @@ public ContinuationInputStream<Long> getRecordsByRecordTypeModifiedAfterLocalTre
             String message = String.format(Locale.ROOT,
                     "getRecordsModifiedAfterLocalTreeJSON(origin='%s', recordType='%s', mTime=%d, maxRecords=%d): " +
                             "Unable to construct URI",
-                    origin, recordType, mTime, maxRecords);
+                            origin, recordType, mTime, maxRecords);
             log.warn(message, e);
             throw new InternalServiceException(message);
         }
 
         log.debug("Opening streaming connection to '{}'", uri);
-        return ContinuationInputStream.from(uri, Long::valueOf);
+        Map<String, String> requestHeaders = new HashMap<String, String>();
+        String token= (String) JAXRSUtils.getCurrentMessage().get(KBAuthorizationInterceptor.ACCESS_TOKEN_STRING);
```

Comment:
where is the map `requestHeaders` used?

Reply:
Correct. Deleted now. Missing clean up from when before it was automatic handled
in kb-util in the ContinuationInputStream.from() method